_docs/_latest/configure/inventory/gsuite.md

```diff
@@ -70,3 +70,68 @@ If you are running Forseti on GCP and made any changes to the above values,
 you will need to copy the `conf` file to the Cloud Storage bucket. For more
 information, see
 [Moving configuration to Cloud Storage]({% link _docs/latest/configure/general/index.md %}).
+
+## Troubleshooting
+
+If have have problem with getting inventory on GSuite data, you can find what
+errors have happened by running `forseti inventory list|get`, or look at the
+`inventory_index_errors` column in the `inventory_index` table.
+
+Error:
+```
+('invalid_grant: Invalid email or User ID', u'{"error" : "invalid_grant", "error_description" : "Invalid email or User ID"}')
+```
+
+Solution:
```

Comment:
It'd be nice to remind them that they'll have to reload the configuration by running `forseti server configuration reload` if they make any changes to `forseti_conf_server.yaml`.

Reply:
Done, added it at the top.